azure-quantum/azure/quantum/optimization/streaming_problem.py

```diff
@@ -153,6 +184,43 @@ def add_terms(self, terms: List[Term]):
             min_coupling = min(term_couplings)
             self.__n_couplers += sum(term_couplings)
             self.stats["num_terms"] += len(terms)
+
+            for term in terms:
```

Comment:
is this supposed to override the above? (lines 182-186)

Reply:
Yes, good catch; thank you.